docs/recovery-mode.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Recovery Mode
+layout: default
+nav_order: 18
+---
+
+## Recovery Mode
+
+{: .note }
+Your data is unaffected in recovery mode unless you use a recovery image or enter [developer mode](entering-developer-mode.md).
+
+{: .warning }
+Do not continue if your device is currently owned or enrolled by a organization, such as a school or employer. Unless you were given permission by your organization, its a crime.
+
+Recovery Mode is a mode that all chromeOS devices enters when it detects an issue with chromeOS or the firmware that prevents it from booting normally. Used to recover the operating system and firmware. Also can be entered manually. 
```

Comment:
Recovery mode is used to restore the OS and RW regions of the firmware. It gets triggered when an issue with the device is detected, or you manually start it.